Makefile

```
@@ -98,24 +100,38 @@ dev:
 # ========================================
 
 test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (15).

(maxbodylength)